### PR TITLE
Skip running chromatic flow again on reopen of PR

### PR DESCRIPTION
**Pull request by gyfchong**
## Why
Weird edge case where if your chromatic ci step has previously built and passed all tests, and you close/re-open your branch, it will skip the publishing of chromatic (since nothing has changed) causing the subsequent test steps to fail

## What
Remove the "reopened" type of pull request event

### CODE REVIEW

File: .github/workflows/test-stories.yaml [hunk 1]
```diff
@@ -7,7 +7,6 @@ on:
   pull_request:
     types:
       - opened
-      - reopened
```
Comment (HeartSquared) [inline .github/workflows/test-stories.yaml:10]: Hahaha, I suppose this is another option :P

**Review by HeartSquared — APPROVED** [review]
